deploy/prometheus/prometheus.yaml

```diff
@@ -14,12 +14,16 @@ spec:
   serviceMonitorSelector:
     matchLabels:
       service: clickhouse-operator
+  ruleSelector:
+    matchLabels:
+      role: alert-rules
+      prometheus: prometheus
   enableAdminAPI: false
-#  alerting:
-#    alertmanagers:
-#    - namespace: default
-#      name: alertmanager
-#      port: web
+  alerting:
+    alertmanagers:
+    - namespace: ${PROMETHEUS_NAMESPACE}
```

Comment:
I would strictly suggest to rename `prometheus.yaml` into `prometheus-template.yaml`, since it has templated parameters from now